specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2025-11-01-preview/examples/fleet/CosmosDBFleetspaceUpdate.json

```diff
@@ -33,7 +33,9 @@
             "serviceTier": "GeneralPurpose",
             "dataRegions": [
               "westus2"
-            ]
+            ],
+            "dedicatedRUs": 2000,
```

Comment:
Same comment as the create fleetspace. If it is needed, let's make the dedicated vs consumable RU/s the same in each one